DockDoor/Views/Hover Window/WindowPreviewHoverContainer.swift

```diff
@@ -53,17 +53,11 @@ struct WindowPreviewHoverContainer: View {
         ZStack {
             ScrollViewReader { scrollProxy in
                 ScrollView(orientationIsHorizontal ? .horizontal : .vertical, showsIndicators: false) {
-                    DynStack(direction: orientationIsHorizontal ? .horizontal : .vertical, spacing: 16) {
-                        if !minimizedOrHiddenWindows.isEmpty {
-                            minimizedOrHiddenWindowsView
-                        }
-                        
-                        ForEach(activeWindows.indices, id: \.self) { index in
-                            WindowPreview(windowInfo: activeWindows[index], onTap: onWindowTap, index: index,
+                        ForEach(windows.indices, id: \.self) { index in
+                            WindowPreview(windowInfo: windows[index], onTap: onWindowTap, index: index,
```

Comment:
We shouldn't get rid of the DynStack usage here, that is how the window preview renders vertically when the dock is set to left/right.